### DISCUSSION TITLE
Dependency Graph not showing dependents

### DISCUSSION THREAD

**Discussion by aminalaee** [community/community#52606] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I've read multiple discussions but couldn't fix this issue that my project does not show dependent packages, but comparing my configurations I can't see any difference so I think there might be an issue.

My project is:
https://github.com/aminalaee/sqladmin/network/dependents
which doesn't show any dependants and doesn't show `Used By` in repo page.

But at least I have another package which lists the above project as a dependency, and I guess there might be more:
https://github.com/aminalaee/sqladmin-demo/network/dependencies

Can anyone tell me how this can happen?

**Comment by sonnees** [comment]
hi, can you rate this comment as an answer? I'm on a mission :v
